src/main/java/Main.java

```diff
@@ -1,205 +1,148 @@
-import com.microsoft.azure.cosmosdb.*;
-import com.microsoft.azure.cosmosdb.rx.AsyncDocumentClient;
 import net.sf.ehcache.Cache;
 import net.sf.ehcache.CacheManager;
 import net.sf.ehcache.Element;
+
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
-import rx.Observable;
-import rx.Scheduler;
-import rx.schedulers.Schedulers;
 
-import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
-import java.util.Map;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
+
+import com.azure.cosmos.ConsistencyLevel;
+import com.azure.cosmos.CosmosClient;
+import com.azure.cosmos.CosmosClientBuilder;
+import com.azure.cosmos.CosmosContainer;
+import com.azure.cosmos.CosmosDatabase;
+import com.azure.cosmos.implementation.InternalObjectNode;
+import com.azure.cosmos.models.CosmosContainerProperties;
+import com.azure.cosmos.models.CosmosQueryRequestOptions;
+import com.azure.cosmos.models.FeedResponse;
+import com.azure.cosmos.util.CosmosPagedIterable;
 
 
 public class Main {
-    private final ExecutorService executorService;
-    private final Scheduler scheduler;
+    private CosmosClient client;
 
-    private AsyncDocumentClient client;
+    private CosmosDatabase database;
 
     private final String databaseName = "test-kcdb";
 
-    public Main() {
-        executorService = Executors.newFixedThreadPool(100);
-        scheduler = Schedulers.from(executorService);
-    }
-
     public void close() {
-        executorService.shutdown();
         client.close();
     }
 
-
     public static void main(String[] args) {
         Main p = new Main();
```

Comment:
AccountSettings.java doesn't need to exist. It makes it harder for customers to follow the flow and we're pulling in external dependencies that aren't necessary.
It can be deleted replaced with the following lines:

```java
public static void main(String[] args) {
    final String accountKey = System.getProperty("ACCOUNT_KEY", "").trim();
    final String accountHost = System.getProperty("ACCOUNT_HOST", "").trim();

    if (accountKey.isEmpty()) {
        System.err.println("ACCOUNT_KEY is not set");
        return;
    } else if (accountHost.isEmpty()) {
        System.err.println("ACCOUNT_HOST is not set");
        return;
    }

    Example p = new Example();

    try {
        p.getStartedDemo(accountKey, accountHost);
        System.out.println("Demo complete, please hold while resources are released");
    } catch (Exception e) {
        System.err.printf("DocumentDB GetStarted failed with %s%n", e);
    } finally {
        System.out.println("Close the client");
        p.close();
    }
}

private void getStartedDemo(String accountKey, String accountHost) throws Exception {
    System.out.println("Using Azure Cosmos DB endpoint: " + accountHost);

    client = new CosmosClientBuilder()
            .endpoint(accountHost)
            .key(accountKey)
            .consistencyLevel(ConsistencyLevel.SESSION)
            .buildClient();

    database = client.getDatabase(databaseName);

    // Demo CosmosDB Pagination
    queryDocumentsByPage();

    // Demo Querying a Document with a list saved into Cache which can be used sliced in UI code for pagination
    executeSimpleQueryWithList();
}
...
```